testsuite/pytests/test_connect_pairwise_poisson.py

```diff
@@ -0,0 +1,111 @@
+# -*- coding: utf-8 -*-
+#
+# test_connect_pairwise_poisson.py
+#
+# This file is part of NEST.
+#
+# Copyright (C) 2004 The NEST Initiative
+#
+# NEST is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# NEST is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+
+
+import unittest
```

Comment:
@ackurth Could you convert this test to the new style using Pytest? 

For inspiration, see, e.g., [test_multimeter.py](https://github.com/nest/nest-simulator/blob/master/testsuite/pytests/test_multimeter.py), [test_rate_connections.py](https://github.com/nest/nest-simulator/blob/master/testsuite/pytests/sli2py_connect/test_rate_connections.py), [test_spike_generator.py](https://github.com/nest/nest-simulator/blob/master/testsuite/pytests/sli2py_stimulating/test_spike_generator.py) and [test_threaded_spike_recorder.py](https://github.com/nest/nest-simulator/blob/master/testsuite/pytests/sli2py_recording/test_threaded_spike_recorder.py).

Reply:
Hey @nicolossus.
Thanks for the review and the nice comments.
To my understanding of pytest (which is limited, so please correct me if I am wrong) converting the this test would mean to rework the general testing suite for connections.
Currently, the tests for connections rules inherit from ```connect_test_base``` which defines (among other things) universal tests for connection routines.
These, I would argue, one wants to parameterize using the available functionality from ```pytest``` and then have specific scripts for the specific tests of the different connection rules.
The only quick way to achieve the proposed would be to mix ```pytest``` and ```unittest``` which might not be the best idea.
A proper solution would in my understanding require a rewriting of the "test hierarchy" for the connection rules and is, I think, beyond the scope of this PR.
If we aim for this, you can just create an issue and assign me to it.
As soon as I find the time I'd take care of this then.

Reply:
@ackurth You are right, I didn't see the huge effort a conversion would require. A conversion should happen at some point, but I agree it is beyond the scope of this PR.